fastlane_core/lib/fastlane_core/ui/help_formatter.rb

```diff
@@ -0,0 +1,13 @@
+require 'commander'
+
+module FastlaneCore
+  class HelpFormatter < ::Commander::HelpFormatter::TerminalCompact
+    def template(name)
+      if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.6')
+        ERB.new(File.read(File.join(File.dirname(__FILE__), "#{name}.erb")), nil, '-')
+      else
+        ERB.new(File.read(File.join(File.dirname(__FILE__), "#{name}.erb")), trim_mode: '-')
```

Comment:
I'm wondering which directory is being loaded in the the failing case @joshdholtz